index.html

```diff
@@ -77,6 +81,21 @@
 
 
 	</header>
+	<section class="promotion-box">
+		<div class="container">
+			<div class="text-box">
+				<p>Best Furniture For Your Castle....</p>
+				<h1>New Furniture Collection
+					Trends in 2020</h1>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Magna in est adipiscing
+						in phasellus non in justo.</p>
+						<div><button>Shop Now</button></div>
+						<div><img src="./images/icons/slider.svg" alt="slider"></div>
+
+			</div>
+			<div class="images"><img src="./images/sofa.png" alt="sofa"></div>
+		</div>
+	</section>
 
```

Comment:
Delete it. You work only with navigation. This section should be done at other branch that you will create after merging this one